.circleci/images/primary/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN mv composer.phar /usr/local/bin/composer
 RUN composer global require hirak/prestissimo
 
 # Install XDebug.
-RUN pecl install xdebug-2.5.5 \
+RUN pecl install xdebug-2.6.0 \
```

Comment:
Do we even need to specify a version still? I wonder if a simple `pecl install xdebug` would suffice now.

Reply:
good point. PR Updated. Also updating robo version.